torch/_torch_docs.py

```diff
@@ -928,36 +912,36 @@ def merge_dicts(*dicts):
 """)
 
 add_docstr(torch.cholesky_solve, r"""
-cholesky_solve(b, u, upper=False, out=None) -> Tensor
+cholesky_solve(input, input2, upper=False, out=None) -> Tensor
 
 Solves a linear system of equations with a positive semidefinite
-matrix to be inverted given its Cholesky factor matrix :attr:`u`.
+matrix to be inverted given its Cholesky factor matrix :attr:`input2`.
```

Comment:
Could we create an alias for input2 as u and input as b? I feel that it doesn’t look as good in math expressions.

For example:
```..... its Cholesky factor matrix :math:`u` (given by :attr:`input2`) ....```